test/state/block.cpp

```diff
@@ -16,6 +19,35 @@ static constexpr uint64_t MAX_BLOB_GAS_PER_BLOCK_PRAGUE = 1179648;
 static constexpr uint64_t BLOB_GASPRICE_UPDATE_FRACTION_CANCUN = 3338477;
 static constexpr uint64_t BLOB_GASPRICE_UPDATE_FRACTION_PRAGUE = 5007716;
 
+uint64_t calc_base_fee([[maybe_unused]] int64_t block_number, int64_t parent_gas_limit,
+    int64_t parent_gas_used, uint64_t parent_base_fee) noexcept
+{
+    auto parent_gas_target = parent_gas_limit / GAS_LIMIT_ELASTICITY_MULTIPLIER;
+
+    // Special logic for block activating EIP-1559 is not implemented, because test don't cover it.
+    assert(block_number != 12'965'000);
```

Comment:
not sure if good idea

Reply:
Just leave the comment, but remove the assert.